tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -232,3 +235,22 @@ def test_nodereplacement_proactive(self):
         assert ceph_cluster_obj.wait_for_rebalance(
             timeout=1800
         ), "Data re-balance failed to complete"
+
+
+@tier4b
+@skipif_no_lso
+@bugzilla("1840539")
+class TestNodeReplacementLSO(ManageTest):
+    """
+    Node replacement twice on LSO:
+    node_1 -> node2
+    node2 -> node1
+    """
+
+    def test_nodereplacenet_lso(self):
+        osd_node_names = node.get_osd_running_nodes()
+        osd_node_name = random.choice(osd_node_names)
+        log.info(f"Selected OSD is {osd_node_name}")
+
+        new_node_name = node.delete_and_create_osd_node_vsphere_upi_lso(osd_node_name)
```

Comment:
How is this function or the functions called by this function going to determine that there is no entry related to the replaced host exists in osd tree?

Reply:
Added node_replacement_verification_steps_ceph_side() call